packages/wallet/ui/src/components/WalletConnection.jsx

```diff
@@ -67,8 +79,70 @@ const WalletConnection = ({
       },
     }).catch(rethrowIfNotCancelled);
 
+    /** @type {Map<string,[ReturnType<typeof makeCapTP>, number]>} */
+    const dappToConn = new Map();
+    const {
+      localStorage: storage,
+      addEventListener,
+      removeEventListener,
+    } = window; // WARNING: ambient
```

Comment:
consider using `window.` at each access so the ambience is apparent at the callsite

Reply:
I'd rather go the other direction: pass in these parts of `window` explicitly rather than using `window` ambiently.

OTOH, `window.addEventListener` _is_ quite likely more conventional and easier to read, and since the current code doesn't follow ocap discipline, maybe making it easier to read is better.

Reply:
Switched to `window.`